all/presets/atari/a2c.py

```diff
@@ -23,6 +23,10 @@ def a2c(
         # Batch settings
         n_envs=16,
         n_steps=5,
+        # Model construction
+        feature_model_constructor=nature_features,
+        value_model_constructor=nature_value_head,
+        policy_model_constructor=nature_policy_head
```

Comment:
Not sure with GitHub is showing these as changes, looks okay in the file view, though.